docker/puppet-agent/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM puppet/puppet-agent-ubuntu:5.5.1
+
+LABEL org.label-schema.maintainer="Puppet Release Team <release@puppet.com>" \
+      org.label-schema.vendor="Puppet" \
+      org.label-schema.url="https://github.com/puppetlabs/puppet-in-docker" \
+      org.label-schema.name="Puppet Agent" \
+      org.label-schema.license="Apache-2.0" \
+      org.label-schema.version=$PUPPET_AGENT_VERSION \
+      org.label-schema.vcs-url="https://github.com/puppetlabs/puppet-in-docker" \
+      org.label-schema.vcs-ref="b75674e1fbf52f7821f7900ab22a19f1a10cafdb" \
+      org.label-schema.build-date="2018-05-09T20:10:27Z" \
+      org.label-schema.schema-version="1.0" \
+      org.label-schema.dockerfile="/Dockerfile"
```

Comment:
Is there a reason that all these labels (and perhaps some additional Dockerfile info) can't/shouldn't be shared?

Reply:
Docker doesn't allow that. 

This image is the puppet-agent-ubuntu image with different labels (you can tell the base image from the FROM statement)

Reply:
yeah, I was wondering if certain things could be DRY'd up, but sounds like that's not really possible and potentially not ideal anyway.